src/schemas/json/sarif.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
>$schema [](start = 3, length = 7)

If you tell me that `sarif.json` and `sarif-2.0.0.json` are intended to be identical, I won't attempt to verify that by hand.

Reply:
yes. i copy over 2.0.0 with the other file.

---
In reply to: [215994704](https://github.com/SchemaStore/schemastore/pull/542#discussion_r215994704) [](ancestors = 215994704)